<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>572</width>
    <height>752</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="folderButton">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>11</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>选择文件夹</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>46</y>
     <width>391</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>请输入要查找的关键词（多个关键词以英文','进行分隔）</string>
   </property>
  </widget>
  <widget class="QListWidget" name="result_list">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>210</y>
     <width>551</width>
     <height>192</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>178</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="showcontentButton">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>420</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>显示内容</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>460</y>
     <width>156</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>是否打开pdf并高亮</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>490</y>
     <width>551</width>
     <height>251</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>68</y>
     <width>551</width>
     <height>101</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>190</y>
     <width>90</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>搜索结果如下</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
